src/dft.cpp

```diff
@@ -338,17 +367,23 @@ void load_dft_hdf5(dft_chunk *dft_chunks, component c, h5file *file, const char
 dft_flux::dft_flux(const component cE_, const component cH_, dft_chunk *E_, dft_chunk *H_,
                    double fmin, double fmax, int Nf, const volume &where_,
                    direction normal_direction_, bool use_symmetry_)
-    : Nfreq(Nf), E(E_), H(H_), cE(cE_), cH(cH_), where(where_), normal_direction(normal_direction_),
+    : E(E_), H(H_), cE(cE_), cH(cH_), where(where_), normal_direction(normal_direction_),
+      use_symmetry(use_symmetry_) {
```

Comment:
Can't you use the initializer for this?   i.e. have `freq(linspace(fmin, fmax, Nf)) { }`